### ISSUE TITLE
Release update that fixes servo jitter issue on R4

### ISSUE THREAD

**Issue by bfinio** [arduino-libraries/Servo#128]
Hi,

I'm coming from this thread in the Arduino forums: https://forum.arduino.cc/t/trouble-with-servos-on-r4-wifi/1151749/10

TLDR there is an issue with servo jitter on the R4 as of library version 1.2.1, it looks like the issue has been fixed and you can do a manual update, but that fix is not part of the official release yet. Is there any plan/timeline for the next official release? I am new to GitHub so I have absolutely no idea where this is on a spectrum of "sure, we can just push that release out now" to "no, there are two dozen other things we need to do first and then we're going to bundle them all in the next release."

For context, I work for a K-12 STEM education nonprofit and we try to keep things as seamless as possible/low-barrier-to-entry for our users, e.g. just using the IDE as-is without manually installing any libraries. Right now we are sticking with the R3 in our Arduino kits because of this issue with the R4, since many of our projects use servos. 

Thanks!

Ben Finio
Science Buddies

**Comment by tomoyanonymous** [comment]
It looks like the code is merged in v1.2.2. https://github.com/arduino-libraries/Servo/releases/tag/1.2.2
